workshops/evals/evals.ipynb

```diff
@@ -0,0 +1,757 @@
+            "metadata": {},
+            "source": [
+                "### Datasets\n",
+                "\n",
+                "Inspect includes some in-built datasets.\n",
+                "We use a dataset that evaluates an LLMs understanding of theory of mind.\n",
+                "\n",
+                "You can view the raw jsonl file where the dataset is stored on Inspect's GitHub repo at the link\n",
+                "- https://github.com/UKGovernmentBEIS/inspect_ai/blob/main/src/inspect_ai/dataset/_examples/theory_of_mind.jsonl\n",
+                "\n",
+                "For those interested, the history of the dataset can be traced through these three repos:\n",
+                "- https://github.com/openai/evals/tree/main/evals/registry/data/theory_of_mind/tomi\n",
+                "- https://github.com/facebookresearch/ToMi\n",
+                "- https://github.com/kayburns/tom-qa-dataset"
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "# load the dataset and see how many samples it has\n",
+                "dataset = example_dataset(\"theory_of_mind\")\n",
+                "print(len(dataset))"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "We see this dataset has 100 samples.\n",
+                "Let us look at an individual sample."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "# you can change the index to see different samples\n",
+                "sample = dataset[2]\n",
+                "\n",
+                "print(sample)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "We see that sample objects in Inspect hold a variety of information.\n",
+                "The most important are the input and target.\n",
+                "The following cell extracts the raw strings to make it easier to view."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "# obtain the underlying raw strings for the input and target and store\n",
+                "# in a variables call input_string and target_string\n",
+                "\n",
+                "# Hide: hard\n",
+                "input_string = sample.input[0].content\n",
+                "target_string = sample.target\n",
+                "# Hide: none\n",
+                "\n",
+                "print(input_string)\n",
+                "print()\n",
+                "print(target_string)"
+            ]
```

Comment:
This dataset seems pretty bad? It's not always correct English, and it's sometimes not precise enough that I can find the answer. Is it normal? If so, we should at least have one sentence about it.

Reply:
which dataset is this? the theory of mind or deception?

Reply:
I am guessing this is the theory of mind dataset.  I have not looked carefully at it but dont remember seeing incorrect English and tasks which are not precise enough.  Worse problem is there are actually mistakes in it (inherited from openai evals - aisi just copied first 100 samples from there). see https://github.com/UKGovernmentBEIS/inspect_ai/issues/66

Reply:
I expected better from them :(